src/libs/actions/IOU.js

```diff
@@ -870,6 +870,8 @@ function createDistanceRequest(report, participant, comment, created, category,
     // If the report is an iou or expense report, we should get the linked chat report to be passed to the getMoneyRequestInformation function
     const isMoneyRequestReport = ReportUtils.isMoneyRequestReport(report);
     const currentChatReport = isMoneyRequestReport ? ReportUtils.getReport(report.chatReportID) : report;
+    const currentTime = DateUtils.getDBTime();
+    const currentCreated = created === format(new Date(currentTime), CONST.DATE.FNS_FORMAT_STRING) ? currentTime : created;
```

Comment:
Isn't this equivalent to [isSameDay](https://date-fns.org/v3.2.0/docs/isSameDay)?